content/docs/vmclarity/getting-started/deploy-azure/_index.md

```diff
@@ -16,41 +16,39 @@ weight: 110
 
 1. Click [here](https://portal.azure.com/#blade/Microsoft_Azure_CreateUIDef/CustomDeploymentBlade/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fopenclarity%2Fvmclarity%2Fmain%2Finstallation%2Fazure%2Fvmclarity.json/uiFormDefinitionUri/https%3A%2F%2Fraw.githubusercontent.com%2Fopenclarity%2Fvmclarity%2Fmain%2Finstallation%2Fazure%2Fvmclarity-UI.json) to deploy VMClarity's custom template.
 
-2. Fill out the required project and instance details in the Basics tab.
+2. Fill out the required **Project details** and **Instance details** in the **Basics** tab.
 
-    <p align="center" width="100%">
-        <img width="50%" src="azure-template-basics.png">
-    </p>
+    ![Set the basic parameters of your VMClarity deployment](azure-template-basics.png)
```

Comment:
I was using paragraph here to be able to control the size of the screenshot. The not scaled version looked way too big in my opinion, what do you think?